src/util/scroll.js

```diff
@@ -124,6 +124,11 @@ function scrollToPosition (shouldScroll, position) {
   }
 
   if (position) {
-    window.scrollTo(position.x, position.y)
+    const docEl: any = document.documentElement
+    if ('scrollBehavior' in docEl.style) {
+      window.scrollTo({ top: position.y, left: position.x, behavior: isObject && typeof shouldScroll.behavior === 'string' ? shouldScroll.behavior : 'auto' })
```

Comment:
let's do

```js
behavior:  (isObject && shouldScroll.behavior) || 'auto'
```

the user should be responsible for passing the correct type of value